sdks/typescript/packages/sdk/package.json

```
@@ -42,10 +59,17 @@
     "node": ">=18.0.0"
   },
   "peerDependencies": {
+    "react": "^18.0.0 || ^19.0.0-0 || ^19.0.0",
```

```suggestion
    "react": ">=18.0.0",
```

I think that's enough? I'm not sure :thinking: